cms/models.py

```diff
@@ -215,17 +215,22 @@ class CoursesInProgramPage(CourseProgramChildPage):
     CMS Page representing a "Courses in Program" section in a program
     """
 
-    # We need this to be only under a program page
-    parent_page_types = ["ProgramPage"]
+    # We need this to be only under a program page and home page
+    parent_page_types = ["ProgramPage", "HomePage"]
```

Comment:
same with this model name, because models names are not reflecting what actually we are doing.

Reply:
Please refer to my answer on your previous comment.